scripts/configure-gha-image-publishing

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# This is needed because it's difficult (if not impossible) to configure a GitHub Action workflow to
+# pass environment variables from a file and refer to them as Docker build args.
```

Comment:
It seems like you should be able to do the following:

```yaml
        - name: Setup Environment Variables                                                                                                                                  
          run: |                                                                   
            echo "ARCH=${{ matrix.arch }}" >> "$GITHUB_ENV"                        
            echo "RKE_VERSION=$(grep -m1 'github.com/rancher/rke' go.mod | awk '{print $2}')" >> "$GITHUB_ENV"
            source  scripts/export-config
            echo $CATTLE_RANCHER_WEBHOOK_VERSION=CATTLE_RANCHER_WEBHOOK_VERSION >> "$GITHUB_ENV"
            echo $CATTLE_CSP_ADAPTER_MIN_VERSION=CATTLE_CSP_ADAPTER_MIN_VERSION >> "$GITHUB_ENV"
            echo $CATTLE_FLEET_VERSION=CATTLE_FLEET_VERSION >> "$GITHUB_ENV"
....
        - name: Build and export server                                                                                                                                       
          id: build                                                                
          uses: docker/build-push-action@v5                                        
          with:                                                                    
            push: false                                                            
            build-args: |                                                          
              "VERSION=${{ env.COMMIT }}"                                          
              "COMMIT=${{ env.COMMIT }}"                                           
              "RKE_VERSION=${{ env.RKE_VERSION }}"                                 
              "ARCH=${{ matrix.arch }}"                                            
              "CATTLE_RANCHER_WEBHOOK_VERSION=${{ env.CATTLE_RANCHER_WEBHOOK_VERSION }}"
              "CATTLE_CSP_ADAPTER_MIN_VERSION=${{ env.CATTLE_CSP_ADAPTER_MIN_VERSION }}"
              "CATTLE_FLEET_VERSION=${{ env.CATTLE_FLEET_VERSION }}"  
```

Similar to how it's currently being done for the RKE_VERSION variable.

Reply:
That way no need to add another generator script for this, we can just re-use the one that creates the export-script file.

Reply:
This is so much better, thank you! Updating...